src/compas_fab/robots/tool.py

```diff
@@ -15,19 +15,32 @@
 class Tool(Data):
     """Represents a tool to be attached to the robot's flange.
 
-    Attributes
+    Properties
     ----------
     visual : :class:`compas.datastructures.Mesh`
         The visual mesh of the tool.
     frame_in_tool0_frame : :class:`compas.geometry.Frame`
-        The frame of the tool in tool0 frame.
+        The tool coordinate frame (TCF) of the tool relative to the planner coordinate frame (PCF).
```

Comment:
Please don't introduce new frame names in this PR, this is about a much simpler issue.

Reply:
Okay, changed back to T0CF